server/service/handler.go

```diff
@@ -131,7 +131,7 @@ func MakeHandler(
 			setRequestsContexts(svc),
 		),
 		kithttp.ServerErrorHandler(&errorHandler{logger}),
-		kithttp.ServerErrorEncoder(encodeErrorAndTrySentry(config.Sentry.Dsn != "")),
+		kithttp.ServerErrorEncoder(encodeError),
```

Comment:
This is the main place where errors were missed in Sentry, I believe. The `ServerErrorEncoder` is a function that gets called when an endpoint returns an error, but we almost never do that, we always return a `nil` error in our endpoints and instead return a struct that implements the `errorer` interface, which we then render as an error. So the fact that we now capture errors in Sentry in the same places as we do for APM (in `ctxerr.Handle`) means that we should collect a lot more entries.